src/utils/domUtils.ts

```diff
@@ -20,13 +20,13 @@ export module Utils {
       return bbox;
     }
 
-    export var POLYFILL_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
+    export var REQUEST_ANIMATION_FRAME_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
```

Comment:
`RenderPolicies.Timeout` uses this value as the default value, and that mainly is the thing that's blocking from hiding this variable.  After thinking about it, I think it's ok to expose this variable